Controller/Callback/Offsite.php

```diff
@@ -214,6 +215,9 @@ public function execute()
                     case Ocbcpao::CODE:
                         $dispPaymentMethod = "OCBC Pay Anyone";
                         break;
+                    case Grabpay::CODE:
+                        $dispPaymentMethod = "GrabPay";
```

Comment:
I see if we add a payment method then we need to add a display method as well. For future improvements, we can extract display payment method text to individual payment class under config and have a factory class that will return a class instance according to the payment code. Then we can simply call `$class->getDisplayPaymentMethod()`.

Reply:
yes I agree, that's a good idea. Maybe we can have a PR to refactor this, including the older payment methods as well. Will keep that in mind 👍🏽